frontend/src/components/ResultsCounter/ResultsCounter.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { getRecipes } from '../../lib/apiClient';
+import './ResultsCounter.css';
+
+
+class SearchResults extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recipesFound: [],
+      searchTerm: null,
+    };
+  }
+
+  componentDidMount() {
+    const { searchTerm } = this.props;
+    this.recipeData(searchTerm);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { searchTerm } = this.props;
+    if (searchTerm !== nextProps.searchTerm) {
+      this.recipeData(nextProps.searchTerm);
+    }
+    this.setState({
+      searchTerm: true,
+    });
+  }
+
+  recipeData(searchTerm) {
+    getRecipes(searchTerm)
+      .then((data) => {
+        this.setState({
+          recipesFound: data,
+        });
+      });
+    if (!searchTerm) {
+      this.setState({
+        searchTerm: false,
+      });
+    }
+  }
```

Comment:
If we moved this into the `RecipeSearchList` component, we could just pass in the list of recipes (or the length) and remove all of these lifecycle methods/`recipeData`.